### ISSUE TITLE
dependency issues from a fresh bundle install

### ISSUE THREAD

**Issue by eatdust** [ibrado/jekyll-paginate-content#1]
Hi there,
cannot make it starts right now, I am clueless. I am running jekyll-3.6.2 and ruby 2.2.8. Any ideas?
Thanks!

```
bundle exec jekyll serve
```
crashes with a dependency error

```
Dependency Error: Yikes! It looks like you don't have jekyll-paginate-content or one of its dependencies installed. In order to use Jekyll as currently configured, you'll need to install this gem. The full error message from Ruby is: 'cannot load such file -- jekyll-paginate-content' I
jekyll 3.6.2 | Error:  jekyll-paginate-content
```

although it is specified in my Gemfile --->
```
group :jekyll_plugins do
  gem "jekyll-feed", "~> 0.6"
  gem "jekyll-gist"
  gem "jekyll-paginate-v2"
  gem "jekyll-paginate-content"
  gem "jekyll-scholar"
end
```
My _config.yml contains:
```
plugins:
  - jekyll-feed
  - jekyll-gist
  - jekyll-paginate-v2
  - jekyll-scholar
  - jekyll-paginate-content
```

and of course I have installed it by running:
```
bundle install
```

as reported by

```
bundle info jekyll-paginate-content
```
which returns:
```
 * jekyll-paginate-content (1.0.3)
        Summary: Jekyll::Paginate::Content: Easily split Jekyll pages, posts, etc. into multiple URLs
        Homepage: https://github.com/ibrado/jekyll-paginate-content
        Path: /home/chris/.gem/gems/jekyll-paginate-content-1.0.3
```

**Comment by ibrado** [comment]
Hi @eatdust,

Try removing `jekyll-paginate-content` from your _config.yml. It should work then.

**Comment by eatdust** [comment]
yes, it starts, thanks!! Time to play :)

**Comment by ibrado** [comment]
Actually, including `jekyll/paginate/content` (with slashes instead of dashes) in _config.yml would have worked. This has been bothering me, though, so to avoid this issue in the future, I've released 1.0.4 that fixes this. Thanks for the report!
